cli/cmd/root.go

```diff
@@ -86,7 +86,8 @@ var rootCmd = &cobra.Command{
 			if err := cli.WaitForVolumeAttachment(abortCtx); err != nil {
 				err = fmt.Errorf("failed to wait for block device being attached: %w", err)
 				if e := cli.MarkDone(ctx, []error{err}); e != nil {
-					logger.Error(fmt.Errorf("failed to update scan result stat to completed with errors: %w", e))
+					//nolint:stylecheck
+					logger.Error(fmt.Errorf("Failed to update scan result stat to completed with errors: %w", e))
```

Comment:
why you need the `//nolint:stylecheck`?
why not just 
```
logger.Errorf("Failed to update scan result stat to completed with errors: %v", err)
```

you don't need to wrap the error since it is not returned, right?

Reply:
Will fix.